src/components/POLine/POLineVersionView/DonorsVersionView/DonorsVersionView.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+import {
+  useCallback,
+  useContext,
+  useMemo,
+} from 'react';
+
+import {
+  DonorsListContainer,
+  VersionViewContext,
+} from '@folio/stripes-acq-components';
+import { TextLink } from '@folio/stripes/components';
+
+import { getDonorUrl } from './utils';
+
+export const DonorsVersionView = ({ version }) => {
+  const donorOrganizationIds = version?.donorOrganizationIds;
+  const versionContext = useContext(VersionViewContext);
+
+  const changedDonorIds = useMemo(() => {
+    const changedDonors = versionContext?.changes?.filter((change) => {
+      return change.path.startsWith('donorOrganizationIds');
+    });
+    const donorIds = changedDonors.map(({ values }) => values.filter(Boolean));
+
+    return donorIds.flat();
+  }, [versionContext?.changes]);
+
+  const renderKeyValueComponent = useCallback(({ value, id, asLink = false }) => {
+    const content = asLink ? <TextLink to={getDonorUrl(id)}>{value}</TextLink> : value;
+    const isUpdated = changedDonorIds?.includes(id);
+
+    const displayValue = isUpdated ? <mark>{content}</mark> : content;
```

Comment:
does this mark tag use default css?
How does it look? Could you add a screencast for the same? Is it in alignment with Folio UX system?

Reply:
@alisher-epam 
Thank you for adding a screen cast.
Are you sure that this yellow color highlight is in line with Folio UX? If not, please have a check with Kimie.